setup.cfg

```diff
@@ -12,14 +12,14 @@ addopts =
     --ignore examples
     --ignore docs
     --doctest-modules
-    --durations 10
-    --timeout 600
-    --cov sktime
-    --cov-report xml
-    --cov-report html
-    --showlocals
-    --matrixdesign True
-    -n auto
+;    --durations 10
+;    --timeout 600
+;    --cov sktime
+;    --cov-report xml
+;    --cov-report html
+;    --showlocals
+;    --matrixdesign True
+;    -n auto
```

Comment:
is this PR meant to be changing this file?

Reply:
Ah my mistake. I have to disable this to get pytest working on my local pc so Ill revert this